ethereum/package-lock.json

```diff
@@ -0,0 +1,12486 @@
+{
```

Comment:
Is this file committed intentionally? I believe we should choose to either use npm or yarn, but not both, since that could lead to undeterministic builds, depending on the build system.